examples/README.md

```diff
@@ -0,0 +1,109 @@
+            
+            If the value reported by the application API is 150:
+            
+                The csp-billing-adapter will report 0 for dimension 1 and 2 and report 150 for dimension 3.
+            
+            If the application API reports 36:
+
+                The csp-billing-adapter will report 0 for dimension 1, 36 for dimension 2, and 0 for dimension 3.
+            
+            In this case the application API may report only 1 dimension. If the application API reports more than 1 dimension it is an error condition.
+
+        The tiered value triggers the code to report per dimension. This has 2 implied reporting mechanisms.
+
+            One use case is the application API reports dimensions that match the configuration.
+
+                In this case, the csp-billing-adapter reports the count per dimension as obtained by the application API.
+
+            The second use case is for the application API to return only 1 value
+
+                In this case the code will divide the reported value into categories reporting for each dimension the maximum value possible. For example:
+                
+                - There are 3 dimensions with a maximum setting of 30, 40, and 55
+                - A minimum setting of 5, 31, and 41
+                
+                The application reports a unit count of 150:
+                
+                    The csp-billing-adapter will report 30 units for dimension 1, 10 units for dimension 2, and 110 units for dimension 3.
+                   
+                The application API reports 36:
+
+                    The csp-billing-adapter will report 30 units for dimension 1 and 6 units for dimension 2, and 0 units for dimension 3.
+
+    usage_aggregation:
+        The value the csp-billing-adapter reports. The maximum over the reporting time period, the current value, or the average over the reporting time period:
+
+            maximum
+                When set to this value, the billing adapter queries the application API hourly and stores the retrieved information in the internal data structure. The csp-billing-adapter stores the maximum value. When the billing adapter reports to the CSP API, the csp-billing-adapter retrieves this value and this is the reported number of units. The internal counter is reset to 0.
+
+            current
+                When set to this value, the billing adapter does not need to store any data from the application, it can query the application API and then directly report the value to the CSP API.
+
+            average
+                When set to this value, the billing adapter queries the application API hourly and stores the retrieved information in he internal data structure. When the billing adapter reports to the CSP API the average over all values is calculated and this is the number of units being reported. The internal storage structure is cleared. Expressed differently sum_of_all_numbers_in_array / length_of_array
+    
```

Comment:
s/he/the/

It was also agreed with PM that we will always round up to the nearest int unit. We should include this rounding in the description.